test/middleware/webhook-timeout.test.js

```diff
@@ -5,7 +5,7 @@ describe('Webhook Timeout', () => {
     const context = {};
     await webhookTimeout(async () => { await sleep(3); }, 1)(context);
     expect(context.timedout).toBeGreaterThan(0);
-    expect(context.timedout).toBeLessThan(15);
+    expect(context.timedout).toBeLessThan(100);
```

Comment:
Timing was all over the place on my box, this just gives the test more headroom. I'm not sure we should be testing that it's less than anything, since I'm not sure what that buys us (and jest times out after 5s) but I didn't want to remove it.

Reply:
I'm working on revamping these assertions so we don't sleep at all. But I would be fine with us dropping this particular assertion. You're right it doesn't really give us much confidence and it's brittle.